README.md

```diff
@@ -1 +1,70 @@
-# catalog-of-my-things
\ No newline at end of file
+# Catalog of our things
+Using OOP, we created a catalog where the user can add books, games and music albuns, having as properties genre, author, and label. It is possible to display a list of all elements by the UI displayed on the console. We followed the following UML diagram structure to also create a schema.sql:
+
+![image](https://user-images.githubusercontent.com/42799579/188950909-66f7bb00-dc45-47e8-8457-cf5669716c8d.png)
+
+## Built With:
+
+- Ruby
+- SQL
+
+## Video Presentation:
+[Check our video presentation about the project](https://drive.google.com/file/d/1JNt88GczT7MJ81UHPq71Oq4qXjrkJk5_/view).
+
+## Getting Started:
+To access and change the code on your computer (using GIT):
+
+> Note: if you don't use git, you can download the ZIP of this repository by clicking on the "Code" button above the files list.
+
+### Setup
+
+- Open the repository [link](https://github.com/Gopxfs/catalog-of-my-things).
+
+- Click on the "Code" button, above the files list.
+
+- Select one of the three options to copy the URL: HTTPS, SSH key and GitHub CLI.
+
+- Open the Terminal in the directory you want to copy the project.
+
+- Type git clone, paste the URL you copied, and press enter.
+

```

Comment:
- [ ] [REQUIRED] Please replace the `Getting started` part with the template I've shown below, or remove that part and put the `Setup` part inside it properly because you haven't provided complete information on how to start your project!

''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''

## `Getting Started`

To get a local copy of this project:

Clone this repository or download the Zip folder:
```
git clone https://github.com/USERNAME/PROJECT_NAME.git
```

To get started, In the root directory run:
```
gem install bundler
```
Then run:
```
bundle install
```
run ruby app in the Command Line:
```
ruby main.rb
```

''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''

Reply:
Thanks for this, updated

Reply:
Well resolved 👍🏽